src/js/components/CheckBox/doc.js

```diff
@@ -27,6 +27,14 @@ export const doc = CheckBox => {
       with the same name so form submissions work.`,
       )
       .defaultValue(false),
+    fill: PropTypes.bool
+      .description(
+        `Whether the checkbox and label expand to fill all of the available
+         width and/or height of their container. When fill === true,
```

Comment:
I don't think we need this last sentence as I think it's describing the internals more than we usually do.